### ISSUE TITLE
Could you tell me optimizer or learning rate settings about CIFAR100?

### ISSUE THREAD

**Issue by DoyoungYoon** [AaronHeee/MEAL#10]
Thank you for sharing your project.

I'm trying this code, but it is missing implement about CIFAR-100 datasets. so could you tell me experiments settings about CIFAR-100? or could you share pretrained model? thank you.

**Comment by AaronHeee** [comment]
Hi, do you mean the pre-trained (teacher) models for CIFAR-100? If so, the training settings is as the same as CIFAR-10 in our experiments, except changing output label from 10 to 100. 

If more questions, please feel free to ask us. Thanks!
